shared/js/background/chrome-events.es6.js

```diff
@@ -56,10 +57,26 @@ chrome.webRequest.onHeadersReceived.addListener(
             // returns a promise
             return ATB.updateSetAtb(request)
         }
+
+        // Strip 3rd party response header
+        const tab = tabManager.get({ tabId: request.tabId })
+        if (!tab || !request.responseHeaders || tab.site.whitelisted) return
+        if (cookieConfig.isExcluded(request.url)) return
+        const index = request.responseHeaders.findIndex(header => { return header.name.toLowerCase() === 'set-cookie' })
+        if (index !== -1) {
+            if (!utils.isFirstParty(request.url, tab.url)) {
```

Comment:
nit: any reason why we are doing this check here and not after `if (cookieConfig.isExcluded(request.url)) return`? It seems like we can do it before we start looking for a header.